modules/tiff/README.md

```diff
@@ -1,7 +1,7 @@
-# @loaders.gl/textures
+# @loaders.gl/tiff
```

Comment:
Nit: If we have both zarr and tiff loaders in this module, there might be a better name; maybe `@loaders.gl/array`? We could also move the `NPYLoader` from its temporary location in `@loaders.gl/terrain` to this new module.

Edit: Didn't realize `NPYLoader` now resided in `@loaders.gl/textures`. Here with TIFF and Zarr might still be a better option.

Reply:
I guess one question is; do we care strongly about the bundle size of this module for script usage when it won't be tree shaken?

Reply:
Agreed that `@loaders.gl/array` could be a good move. We might be able to unify interfaces for in-memory multi- dimensional/scale arrays.

Reply:
I already split into `modules/tiff` and `modules/zarr`

Reply:
> We might be able to unify interfaces for in-memory multi- dimensional/scale arrays.

Maybe we should spend some time discussing a consistent output format first for zarr, tiff, and npy, similar to what we're doing in https://github.com/visgl/loaders.gl/issues/1430

Reply:
I don't have a strong preference on whether tiff and zarr should be in one module or two, though I do feel that the NPYLoader is closer to both tiff and zarr than it is to the other special GPU texture formats in `@loaders.gl/textures`.

Reply:
Seems different to me. The tiff and zarr modules provide a tile source interface, whereas the numpy loader decodes a single tile?